app/controllers/application_controller.rb

```diff
@@ -52,6 +52,7 @@ class ApplicationController < ActionController::Base
   include_concern 'TreeSupport'
   include_concern 'SysprepAnswerFile'
   include_concern 'ReportDownloads'
+  include_concern 'Show'
```

Comment:
I think it's a good direction, I am just concerned that currently it might override/influence other places without intention. But I might be wrong on this. @martinpovolny @himdel  wdyt?